### PR TITLE
adding the login page

### PR DESCRIPTION
**Pull request by atnomoverflow**
The title sais it all

### CODE REVIEW

File: ui/src/pages/Sign-up/sign-up.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                        ignore:"false",
+                    },
+                }
+            ]
+        }
+    },
+    computed: {},
+    methods: {}
+})
+</script>
```
Comment (Klaven) [inline ui/src/pages/Sign-up/sign-up.vue:66]: should these `.vue` files end with a new line? Im torn on how I feel about this. 
https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

part of me would argue that it makes no difference. in the future maybe we try to ensure they all have newlines but I think it's borderline nit... and I tend to think nits don't have to be fixed.

File: ui/src/pages/Sign-up/sign-up.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.input-border {
+    border: 3px solid rgba(66, 153, 225, 0.5);
```
Comment (Klaven) [inline ui/src/pages/Sign-up/sign-up.css:2]: tbh i'm a little confused? I thought we where using tailwind? it's probably me missing something so :) don't worry about me.

**Review by Klaven — APPROVED** [review]
/lgtm
I also posted a few questions so I will not merge it but none of my questions should block it from merging. just wanted you to see them :)

File: ui/src/pages/Sign-up/sign-up.vue (hunk 1 above)
Comment (atnomoverflow) [reply ui/src/pages/Sign-up/sign-up.vue:66]: It realy does not matter we can configure prettierrc file to format the code with new line or what not i think

File: ui/src/pages/Sign-up/sign-up.css (hunk 2 above)
Comment (atnomoverflow) [reply ui/src/pages/Sign-up/sign-up.css:2]: there are some colors that are not supported in tailwind i can add them in config file then extend the colors but if am using it in one place i usally just put it in css if use it more than once then i extend it .

**Pull request merged**